src/annotation/annotations/alias.js

```diff
@@ -14,6 +14,7 @@ export default function alias (env) {
 
         let alias = item.alias
         let name = item.context.name
+        let aliasGroup = item.group
 
         let aliasedItem = Array.find(data, i => i.context.name === alias)
```

Comment:
This lookup finds the **_first_** instance of the alias name, which could be incorrect if there's more than one context with the same name.